src/ResourceManagement/RecoveryServices.Backup/Microsoft.Azure.Management.RecoveryServices.Backup/Customizations/Client.Customizations.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Azure.Management.RecoveryServices.Backup.Models;
```

Comment:
Regarding the comment about generate.cmd, can you give more clarity about which commit should it be? Also, we've tried to follow the Compute project as they've just specified the link to the composite JSON spec: https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/generate.cmd

Why is this guideline different?